src/libraries/Microsoft.PowerFx.Connectors/Shared/ServiceFunction.cs

```diff
@@ -0,0 +1,413 @@
+﻿// <autogenerated>
```

Comment:
Add a comment that this isn't actually autogenerated, it's just to disable stylecop warnings.